src/resources/js/tickets.js

```diff
@@ -337,6 +339,31 @@ var ticketHeaderImage = window.ticketHeaderImage || {};
 			} );
 		}
 
+		function show_hide_panel( $panel, direction ) {
+			var anim = {};
+
+			if ( 'absolute' !== $panel.css( 'position' ) ) {
+				if ( 'bottom' === direction && $panel === $base_panel ) {
+					anim[direction] = '100%';
```

Comment:
Rinse and repeat for other array vars where the index is a variable or calculation.